### DISCUSSION TITLE
MultiServer Economy

### DISCUSSION THREAD

**Discussion by ZhenkM** [Zrips/CMI#3212] [Q&A]
**Question: **
Hello! yesterday on my server I tried to connect 2 servers to the same database to share, inventories, money, etc.

a game mode, divided into several servers to balance the load.

I used CMI because it allows me to connect both chat and "MSG" in bungee mode, but I ran into the problem that when users send money to each other when they are on different servers, they do not receive the money, but it is lost. the same.

i create a video. 
https://www.youtube.com/watch?reload=9&v=ikT47fXTZyw&feature=youtu.be

users are on different servers called server A and server B

the user from server A sends money to the user who is on server B
the user on server A confirms that I send the money, but the user on server B does not receive the money, does not update, or does anything.

After that, the user who is in server B returns to server A and the money arrives, it is updated in the scoreboard and in his balance the money, but after a few seconds the money is eliminated.

the possible that this can get to work?
This function does not work and will not be implemented?
is it a bug? it's a problem?
hank you very much for taking the time to read, I hope for a response very soon. Happy day.

**Comment by CloudeLecaw** [comment]
maybe this is because the money command was not made to sync between servers
